### ISSUE TITLE
Deployment ID is incorrectly generated for removal and info commands when GCP project ID value is interpolated

### ISSUE THREAD

**Issue by amarthadan** [api3dao/airnode#1546]
The remove, list, and info commands generate the deployment IDs from config files stored in the cloud. They don't run the config files through the interpolation though. When one of the fields used for deployment ID generation (e.g. GCP project ID) requires the interpolation, the resulting deployment ID is incorrect.

We need to with each `config.json` file download also the `secret.env` file and run the interpolation before using the values for deployment ID generation.

Reported in #1545

**Comment by martinkolenic** [comment]
Hi @amarthadan, here's the test report. There's still something wrong with deployment removals.

Commit tested: See screenshot below
![Image](https://user-images.githubusercontent.com/884979/204535371-eef0790a-77cd-4980-ac4b-b98756b5a93e.png)

Test 1: Deploy Airnode with the GCP project ID as secret
Steps: Deploy Airnode using modified coingecko-signed-data example with config.json changed so that the project ID is replaced with a variable loaded from secrets.env (GCP_PROJECT_NAME).
Expected result: Successful deployment
Actual result: Matches expected result
![Image](https://user-images.githubusercontent.com/884979/204535433-b6dc73b5-76ae-4deb-9173-f996bfb6c85b.png)
![Image](https://user-images.githubusercontent.com/884979/204535628-f6e6e451-8570-4a02-bf5c-bf93959a7e8b.png)
![Image](https://user-images.githubusercontent.com/884979/204535680-6b362ab2-9be5-45bc-9d21-9b73e7a92b13.png)

Test 2: List Airnode deployments 
Steps: After deployment, list the existing deployments using the `list` command
Expected result: The GCP deployment having what looks like a valid ID and being listed
Actual result: Matches the expected result
Screenshot:
![Image](https://user-images.githubusercontent.com/884979/204535264-037d1e4b-8162-4da0-96ab-97cca18c5de6.png)

Test 3: Remove deployment using deployment ID generated from interpolated project name
Steps: Run the `remove` command using the deployment ID previously displayed in the list command
Expected result: Successful removal
Actual result: The removal fails and an error message is displayed (see screenshot)
![Image](https://user-images.githubusercontent.com/884979/204535205-657febc7-fd93-45ad-a53b-53c7fa9a9283.png)

**Comment by amarthadan** [comment]
Are you sure you used the same commit for the removal? What deployment ID is in the `receipt.json`?

**Comment by martinkolenic** [comment]
The commit should be the same, see:
![image](https://user-images.githubusercontent.com/884979/204559062-28312980-e964-4307-85a2-3052615f205e.png)

The deployment IDs differ though, as the one in `receipt.json` is `gcpfc24392a` and the one in Deployer is `gcpd2a3fb07`. However, this is the same deployment as I haven't done any other deployment today and you can see the date and time matches too.

**Comment by amarthadan** [comment]
But you're testing a wrong commit.
![Screenshot_20221130_074420](https://user-images.githubusercontent.com/1655405/204726595-7c51f202-e190-4d62-a3fd-7c47f5a54348.png)

**Comment by martinkolenic** [comment]
Honestly, I don't understand how that happened because the first thing I did before starting was `git pull`. I see the difference, must've been a moment of blindness on my part. I'm retesting right now.

**Comment by martinkolenic** [comment]
Hi @amarthadan, it seems there was a conflict when I tried to pull the latest commit and my instance of the repo got stuck on an older one. I resolved this by `git reset --hard`. 

However, after doing so, I repeatedly get an error when trying to run `yarn bootstrap` in the root folder of the repo. I wasn't sure if my config wasn't wrong so I purged my node and yarn installations and reinstalled them. Nevertheless, the error remains and is completely identical. 

It seems that npm/yarn (not sure) thinks I have a local registry (if I understand that correctly) but I don't, see screenshot:


![Image](https://user-images.githubusercontent.com/884979/204745891-3647d7c3-18ce-49f4-bb00-f14ba9a486f5.png)

**Comment by Siegrift** [comment]
Probably the last time you were testing packages locally using verdaccio you forgot to switch back the registry. Double check what is your output of `yarn config get registry`, it should not be localhost. You can set reset it by `yarn config set registry https://registry.yarnpkg.co`. Similarly for NPM: `npm get registry` and set it using `npm set registry https://registry.npmjs.org/`.

**Comment by martinkolenic** [comment]
Thanks @Siegrift, got yarn running again. But there's another issue with the latest commmit (Michal's changes did not seem to persist through Derek's last PR) so we're troubleshooting that with Michal.
